src/Reflection/ReflectionClass.php

```diff
@@ -512,9 +512,9 @@ public function getImmediateConstants(): array
      *
      * @return array<string, scalar|array<scalar>|null>
      */
-    public function getConstants(): array
+    public function getConstantsValues(): array
```

Comment:
IMO, let's drop this completely - let the userland consumer do the `array_map()` operations